src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -1226,21 +1226,26 @@ private ImmutableArray<MethodSymbol> ExplicitlyOverriddenMethods()
             return new MetadataDecoder(moduleSymbol, _containingType).GetExplicitlyOverriddenMethods(_containingType.Handle, _handle, this.ContainingType);
         }
 
-        internal MethodSymbol ExplicitlyOverriddenClassMethod
+        internal override MethodSymbol ExplicitlyOverriddenClassMethod
         {
             get
             {
                 if (IsExplicitClassOverride)
                 {
+                    // If there is a *single* methodimpl entry, we return that.
+                    MethodSymbol result = null;
                     foreach (MethodSymbol method in ExplicitlyOverriddenMethods())
                     {
                         if (method.ContainingType.IsClassType())
                         {
-                            return method;
+                            if (result is { })
+                                return null;
```

Comment:
> return null; [](start = 31, length = 13)

Is this code path covered by a test?

Reply:
Yes; see `PEMethodSymbol_ExplicitlyOverriddenClassMethod_WhenAmbiguous` in a later iteration.

---
In reply to: [424901858](https://github.com/dotnet/roslyn/pull/44025#discussion_r424901858) [](ancestors = 424901858)